base/cvd/cuttlefish/host/commands/cvd/cli/selector/arguments_lexer.cpp

```diff
@@ -206,5 +291,16 @@ Result<std::vector<std::string>> ArgumentsLexer::Preprocess(
   return new_args;
 }
 
+Result<std::deque<ArgToken>> TokenizeArguments(
+    const std::vector<std::string>& args) {
+  ArgumentsLexerBuilder builder;
+
+  std::unique_ptr<ArgumentsLexer> lexer = CF_EXPECT(builder.Build());
+  CF_EXPECT(lexer.get());
```

Comment:
`Build()` should return `Result<ArgumentsLexer>` instead of the pointer, so this line wouldn't be needed.

Reply:
Done.